### PR TITLE
docs(Storybook): add Table In Modal With Inline Edit Row story to reproduce button position bug

### PR DESCRIPTION
**Pull request by mturley**
affects: patternfly-react

**What**:

This adds a new story called "Table in Modal With Inline Edit Row", which is just the "Table with Inline Edit Row" story inside the Modal example story.

This exposes a bug with the positioning of the confirm/cancel buttons after clicking the edit button on a row. The buttons appear to be positioned with JS, relative to the window, rather than relative to the modal, the table or the row.

With a large window:

screenshot 2018-08-15 10 50 32

With a small window:

screenshot 2018-08-15 11 59 23

This PR should be merged just to add the story, and then a separate PR should be used to fix the bug..

**Link to Storybook**:

http://localhost:6006/?knob-Show%20Help=true&knob-Field%20Level%20Help%20Content=Please%20specify%20Country%20code%20%3Cbr%3E%20%3Ca%20target%3D%27_blank%27%20href%3D%27https%3A%2F%2Fcountrycode.org%2F%27%3EClick%20here%20for%20a%20list%20of%20Country%20codes%3C%2Fa%3E&knob-Close%20Popover=true&knob-Show%20Modal=true&selectedKind=patternfly-react%2FContent%20Views%2FTable%20View&selectedStory=Table%20in%20Modal%20With%20Inline%20Edit%20Row&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs

**Additional issues**:

This story showcases the bug reported in #535.

### CODE REVIEW

File: packages/patternfly-react/src/components/Modal/__mocks__/mockModalManager.js [hunk 1]
```diff
@@ -27,34 +62,7 @@ export class MockModalManager extends React.Component {
...
-                <label className="col-sm-3 control-label" htmlFor="textInput3">
-                  Field Three
-                </label>
-                <div className="col-sm-9">
-                  <input type="text" id="textInput3" className="form-control" />
-                </div>
-              </div>
-            </form>
-          </Modal.Body>
+          <Modal.Body>{children || defaultBody}</Modal.Body>
```
Comment (AllenBW) [inline packages/patternfly-react/src/components/Modal/__mocks__/mockModalManager.js]: This is a neat idea ❤️

File: yarn.lock [hunk 2]
```diff
@@ -5681,7 +5681,7 @@ execall@^1.0.0:
   dependencies:
     clone-regexp "^1.0.0"
 
-exenv@^1.2.0, exenv@^1.2.1:
+exenv@^1.2.0, exenv@^1.2.1, exenv@^1.2.2:
```
Comment (AllenBW) [inline yarn.lock]: Though this doesn't seem to belong in this pr 🤔

Comment (mturley) [reply yarn.lock]: Hmm, yarn decided to do that and I decided to `git commit -a`... whoops

Comment (mturley) [reply yarn.lock]: Thanks for the review @AllenBW, just rebased and removed this yarn.lock change.
